### PR TITLE
CLOUDSTACK-10024: Network Migration: Tackle review comments

### PR DESCRIPTION
**Pull request by fmaximus**
Tackles review comments given on #2259 after the merge.

Co-Authored-By: Raf Smeets <raf.smeets@nuagenetworks.net>
Co-Authored-By: Kris Sterckx <kris.sterckx@nuagenetworks.net>

### CODE REVIEW

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by krissterckx** [comment]
@rhtyd this is the requested PR - pls review;  open to questions/comments.  Thanks.

Kris

**Comment by rhtyd** [comment]
@krissterckx sure, already did. I'll help kick some tests now, first building packages.

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by rhtyd** [comment]
@fmaximus can you add the new marvin test to Travis (see the .travis.yml file) if it is hardware/hypervisor agnostic and also check/fix the failure in the last run. Thanks.

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
Additional review requested - @borisstoyanov @nvazquez @DaanHoogland @wido @rafaelweingartner @marcaurele and others, thanks.
@blueorangutan package

**Comment by fmaximus** [comment]
@rhtyd Looking at the Trillian logs, the test fails to pick up the VPC offering which was added to test_data.py I double-checked, all test_data keys are correct. 
About Travis, we added code in our tests to make sure to skip the tests if the hypervisor is not supported.
Any preference to where we should add the tests in .travis.yml, not to exceed the time limits even more.

**Comment by rhtyd** [comment]
Thanks @fmaximus Trillian uses it's own test_data. I'll move your test data changes to https://github.com/shapeblue/Trillian/blob/master/Ansible/roles/marvin/templates/test_data.py.j2 before running the next test round. It is advisable to include test data with the test and leave merging changes to the marvin test_data.

@blueorangutan package

File: tools/marvin/marvin/config/test_data.py [hunk 1]
```diff
@@ -419,8 +419,8 @@
         "displaytext": "MySharedOffering",
         "guestiptype": "Shared",
         "supportedservices": "Dhcp,Dns,UserData",
-        "specifyVlan": "True",
-        "specifyIpRanges": "True",
+        "specifyVlan": "False",
```
Comment (rhtyd) [inline tools/marvin/marvin/config/test_data.py]: @fmaximus This test data is in use by several test cases, changing the values from 'true' to 'false' may cause regressions. Can you change them back, and in your tests perhaps change the value to False if necessary?

Comment (fmaximus) [reply tools/marvin/marvin/config/test_data.py]: This *is* the revert of test data, done in PR #2259

Comment (krissterckx) [reply tools/marvin/marvin/config/test_data.py]: @rhtyd this change is exactly the revert you asked for. I think we shd merge in this PR as your comment was legitimate.

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by rhtyd** [comment]
Test LGTM, the new migration test passes as well. I'll merge this based on code reviews and test results.

**Pull request merged**